src/ontology/mondo-edit.obo

```diff
@@ -438962,6 +438965,74 @@ is_a: MONDO:0100355 {source="https://orcid.org/0000-0001-5208-3432"} ! classic o
 property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/3221 xsd:string
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100358
```

Comment:
what is the difference between this term and 'herpes labialis' (MONDO:0043653)
aren't they the same?

Reply:
I'm not sure! Herpes labialis did have the axiom on it - disease has location some lip, then we decided to remove that when we were on the call, because the text def said oralfacial region, and lip was too specific. But maybe they are the same? I'm not sure!

Reply:
per our conversation, I merged these together